### ISSUE TITLE
Running the Camera 

### ISSUE THREAD

**Issue by hellopathere** [sbis04/flutter_camera_demo#2]
Hi,
I get this issue, do you know how to fix it?
<img width="1437" alt="Screen Shot 2021-09-24 at 9 47 58 AM" src="https://user-images.githubusercontent.com/83986056/134686462-8936fada-bfa9-4552-b1ce-58473c775819.png">

**Comment by sbis04** [comment]
@hellopathere Simulator doesn't have access to the camera. If you want to access a camera, you need a device. You can't test the camera functionalities on a Simulator.

**Comment by hellopathere** [comment]
Oh okay that makes sense. So on a real device it would work just like that or do I have to change some code?

**Comment by sbis04** [comment]
Yes, the app will work just like that, you don't have to make any changes. It is tested on both Android and iOS devices.
